### ISSUE TITLE
Chat UI does not calculate correct content offset after fetchMessages

### ISSUE THREAD

**Issue by a821246710** [SkygearIO/chat-SDK-iOS#178]
In `SKYChatConversationViewController`: `fetchMessages ` function end will recalculate contentOffset but I found it has two problems.

case 1:
Does not take `topContentAdditionalInset` into calculation, so if set `topContentAdditionalInset` more than 0 the offset will have problem.

case  2:
Does not take toolbar height into calculation, toolbar will cover part of message if there has enough messages to scroll.

**Comment by ben181231** [comment]
I have gave a PR on the scroll position issue I just found. However, I cannot find any issues mentioned as the description.

The following is the screenshot when user set `SKYChatConversationViewController. topContentAdditionalInset` to `CGFloat(100)`. See if this is an expected behavior.

![img_f95496ecb074-1](https://user-images.githubusercontent.com/3863953/35322959-f76abaa4-0126-11e8-99f7-0ac18ec92a64.jpeg)

The following screencast shows the behavior of conversation view embedded in tab bar controller. See if anything unexpected.

https://youtu.be/2Dlyy76BUIw
